aggregate/aggregation_helpers.py

```diff
@@ -1,4 +1,8 @@
+from matplotlib.pyplot import axis
```

Comment:
@SashaWeinstein @td928 We rneed to organize our helper functions and scripts better but can wait till refactor/not up against deadlines

Reply:
lets discuss where this should all live